### ISSUE TITLE
Port :8008 is occupied. Please close the process that's using it to start the KA Lite

### ISSUE THREAD

**Issue by jiangjianshan** [learningequality/ka-lite#5530]
Hello, 

     I have reinstall my laptop's operating system and installed the KA Lite 0.17.3. When I double click the KA Lite icon on my desktop, one pop hint appear as below.
![image](https://user-images.githubusercontent.com/20179047/32030504-e8b3083a-ba2d-11e7-9362-234451543ff2.png)

      And I have also found the "start server" item on KA Lite tray is still on black color, normally it should be gray color as I remember. So it seems like the sever can't be started.

      Here is the log file I copied from $HOME\.kalite\logs
[django-log.txt](https://github.com/learningequality/ka-lite/files/1416662/django-log.txt)

**Comment by mrpau-richard** [comment]
Hi @jiangjianshan that error means the port 8008 is use by other application. 
Do you have other applications running when starting KA Lite server?
If port 8008 is not available the KA Lite server will not start. To solve that issue, kill or terminate the process that using the port 8008 before running the KA Lite server.

Here's a sample command to kill port process using command prompt before starting KA Lite server.
```
C:\Users\xxx>netstat -ano | findstr :8008
  TCP    0.0.0.0:8008           0.0.0.0:0              LISTENING       7252
  TCP    127.0.0.1:52675        127.0.0.1:8008         TIME_WAIT       0

C:\Users\xxx>taskkill /PID 7252 /F
SUCCESS: The process with PID `7252` has been terminated.
```

**Comment by jiangjianshan** [comment]
@mrpau-richard , thanks for your suggestion. I have just tried your solution but still got the same issue. So I change another way to solve it. I have added one environment variables which name is KALITE_LISTEN_PORT and its value was change from 8008 to 8018. And then I reboot my laptop and start ka-lite again. Now it work.

**Comment by jiangjianshan** [comment]
@mrpau-richard , but I meet another issue is that, ka-lite can't access the internet via port 8018. I think maybe I need to find a available port for it so that ka-lite can access the internet but not use 8008. How to find the available port for internet assess?
 
![image](https://user-images.githubusercontent.com/20179047/32037575-84b25054-ba58-11e7-8a88-c17a59f5e3de.png)

**Comment by jiangjianshan** [comment]
@mrpau-richard , thanks for your solution. It work for me. But I have also tried to use the environment variable KALITE_LISTEN_PORT to set the default port from 8008 to 8080 to KA Lite. But the scenario is I can login into KA Lite but it can't access internet to download the videos. I think I'm using a wrong port settings. Are there other available port can be used for KA Lite?

**Comment by mrpau-richard** [comment]
@jiangjianshan do you have internet access? Your problem in port is already solved. you just need an internet connection to register your KA Lite device.

**Comment by benjaoming** [comment]
Hi @jiangjianshan - please feel welcome in the support community for support questions: http://community.learningequality.org/

Also thanks for reporting the other issue -- we are investigating this with full speed and will make another release to fix it within a few days.
